libbeat/scripts/docker-entrypoint.sh

```diff
@@ -53,7 +53,7 @@ es_url() {
 # green or yellow.
 waitForElasticsearch() {
   echo -n "Waiting on elasticsearch($(es_url)) to start."
-  for ((i=1;i<=60;i++))
+  for ((i=1;i<=120;i++))
```

Comment:
`i` is being incremented twice in the loop. Remove the second `i++` on line 66 and keep it at 60 seconds.